File: Involucrate/Models/Usuario.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Involucrate.Models
{
    public class Usuario
    {
        public int IdUsuario { get; set; }
        public string Nombre { get; set; }
        public string Apellidos { get; set; }
        public string Dni { get; set; }
        public string CorreoElectronico { get; set; }
        public string HashContrasena { get; set; }
        public bool EsONG { get; set; }
    }

}

File: app/src/main/java/com/andrea_murillo/involucrate/model/Usuario.kt
package com.andrea_murillo.involucrate.model

data class Usuario(
    val id: Long? = null,
    val nombre: String,
    val apellidos: String,
    val dni: String,
    val correoElectronico: String,
    val hashContrasena: String,
    val tipo: String,
    val telefono: String,
    val estadoCuenta: String = "activo"
)
